boot/freeldr/freeldr/ntldr/setupldr.c

```diff
@@ -65,6 +65,7 @@ SetupLdrLoadNlsData(PLOADER_PARAMETER_BLOCK LoaderBlock, HINF InfHandle, PCSTR S
 #if DBG
     {
         BOOLEAN Success = WinLdrLoadNLSData(LoaderBlock, SearchPath, AnsiName, OemName, LangName);
+        (VOID)Success;
```

Comment:
Why is this considered as an "unused" variable, since it is used in the TRACE() function call just below? o_O

Reply:
Because freeldr defines it this way for arm:
https://git.reactos.org/?p=reactos.git;a=blob;f=boot/freeldr/freeldr/include/debug.h;hb=e85664a3d8dfb955b56b2f5d27a8ccd85db9454b#l41

Which seems like a hack, but yeah.

Reply:
We may need to do (in freeldr's debug.h) something similar to https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/include/reactos/debug.h;h=ab332fda18145b9de372dfcb4f07209d0afbde6e;hb=HEAD#l134 .